mtags-interfaces/src/main/java/scala/meta/pc/PresentationCompiler.java

```diff
@@ -38,6 +38,16 @@ public abstract class PresentationCompiler {
     // Language Server Protocol APIs.
     // ==============================
 
+
+     /**
+     * Returns token informations from presentation compiler.
+     *
+     */
+    public abstract CompletableFuture<List<Integer>> semanticTokens(
+            VirtualFileParams params, 
+            List<String> capableTypes, 
+            List<String> capableModifiers);
```

Comment:
```suggestion
    public abstract CompletableFuture<List<Integer>> semanticTokens(VirtualFileParams params);
```
as I said above, we don't need to send types and modifiers.